contrib/pkginstaller/package.sh

```diff
@@ -14,13 +14,18 @@ tmpBin="contrib/pkginstaller/tmp-bin"
 
 binDir="${BASEDIR}/root/podman/bin"
 libDir="${BASEDIR}/root/podman/lib"
+docDir="${BASEDIR}/root/podman/docs/man/man1"
 
 version=$(cat "${BASEDIR}/VERSION")
 arch=$(cat "${BASEDIR}/ARCH")
 
 function build_podman() {
   pushd "$1"
 
+  make docs
```

Comment:
this should be `make podman-remote-darwin-docs`, we do some string subs for remote docs.